src/ui/components/results/verticalresultscomponent.js

```diff
@@ -241,7 +242,8 @@ export default class VerticalResultsComponent extends Component {
   getVerticalURL (data = {}) {
     const verticalConfig = this._verticalsConfig.find(config => config.verticalKey === this.verticalKey) || {};
     const verticalURL = this._config.verticalURL || verticalConfig.url || data.verticalURL || this.verticalKey + '.html';
-    return addParamsToUrl(verticalURL, { query: this.query });
+    const dataTabOrder = this.core.globalStorage.getState(StorageKeys.NAVIGATION) ? this.core.globalStorage.getState(StorageKeys.NAVIGATION).tabOrder : [];
+    return addParamsToUrl(verticalURL, { query: this.query, tabOrder: getTabOrder(this._verticalsConfig, dataTabOrder) });
```

Comment:
nit but can we have the getTabOrder call on another line? I think it's a little easier to read that way (not squishing multiple calls into one line and you also get to name the variable something descriptive)